q2_feature_table/_summarize/summarize_assets/src/controls.js

```diff
@@ -41,8 +52,7 @@ const initializeControls = () => {
 
   const updateStats = (trows, slider) => {
     formGroup.selectAll('span')
-      .data([['Samples Retained: ', `${calcSampleRetainment(trows, slider).join(' (')}%)`],
-             ['Sequences Retained: ', `${calcFeatureRetainment(trows, slider).join(' (')}%)`]])
+      .data([['Retained ', `${calcFeatureRetainment(trows, slider).join(' (')}%)`, ' sequences in ', `${calcSampleRetainment(trows, slider).join(' (')}%)`, ' samples at the specifed sampling depth.']])
```

Comment:
I think all of this can be simplified to:
```javascript
const sequences = `${calcFeatureRetainment(trows, slider).join(' (')}%)`
const samples = `${calcSampleRetainment(trows, slider).join(' (')}%)`
fromGroup.selectAll('span')
  .text(`Retained ${sequences} sequences in ${samples} samples at the specifed sampling depth.`)
  .insert('br')